### PR TITLE
Update 2020-08-21-unity-csharp-games-security.md

### PR DESCRIPTION
**Pull request by nickgamb**
Updated Identity Security for Games in C# with Unity to use CLI and remove references to old admin console.

This blog post has the following features:

- [X] A GitHub Repository with a polished README
- [X] A GitHub Repository under the github.com/oktadeveloper account
- [X] A title that's approved by Dev Advocacy
- [X] A URL that's approved by Dev Advocacy
- [X] The content has been ran through Grammarly (https://www.grammarly.com/)
- [X] Rendered locally and confirmed that no Markdown typos exist

### CODE REVIEW

File: _source/_posts/2020-08-21-unity-csharp-games-security.md [hunk 1]
```diff
@@ -36,20 +35,6 @@ The level of effort to build identity security from scratch is massive. Just to
...
-NATIVE: [https://youtu.be/9C5xaaLt7Q4](https://youtu.be/9C5xaaLt7Q4)
-<div style="text-align: center">
-<iframe width="600" height="338" src="https://www.youtube.com/embed/9C5xaaLt7Q4" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-</div>
-
-OAUTH: [https://youtu.be/ciDl0-Wtfvo](https://youtu.be/ciDl0-Wtfvo)
-<div style="text-align: center">
-<iframe width="600" height="338" src="https://www.youtube.com/embed/ciDl0-Wtfvo" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-</div>
-
```
Comment (mraible) [inline _source/_posts/2020-08-21-unity-csharp-games-security.md:52]: Why remove?

File: _source/_posts/2020-08-21-unity-csharp-games-security.md [hunk 2]
```diff
@@ -129,6 +110,10 @@ Next, a new menu will be needed for the Auth Menu. The simplest way to create th
 
 Click `MenuManager` and make sure it is enabled in the inspector. Find the `PanelManager` script component attached to the `MenuManager` object and double click to edit. This will open up the primary IDE configured in Unity.
 
+## Okta CLI
+
+{% include setup/cli.md type="web" loginRedirectUri="http://127.0.0.1:51772" logoutRedirectUri="http://127.0.0.1:51772"%}
```
Comment (mraible) [inline _source/_posts/2020-08-21-unity-csharp-games-security.md:115]:
```suggestion
{% include setup/cli.md type="web" loginRedirectUri="http://127.0.0.1:51772" logoutRedirectUri="http://127.0.0.1:51772" %}
```

File: _source/_posts/2020-08-21-unity-csharp-games-security.md [hunk 3]
```diff
@@ -213,8 +198,7 @@ Make sure `MenuManager` is selected and notice the new properties in the `PanelM
 1. Drag the `AuthMenu` object from the project hierarchy to the `authMenu` property in the inspector.
 2. Drag the InputField from the `Username` object in the project hierarchy to the `Username` property in the inspector.
 3. Drag the InputField from the `Password` object in the project hierarchy to the `Password` property in the inspector.
-4. Enter the Okta domain from the Okta developer org "{yourOktaDomain}".
-
+4. Replace `{yourOktaDomain}` with the domain of the Okta developer org created by the CLI.
```
Comment (mraible) [inline _source/_posts/2020-08-21-unity-csharp-games-security.md:201]:
```suggestion
4. Replace `{yourOktaDomain}` with the domain of your Okta developer org.
```

File: _source/_posts/2020-08-21-unity-csharp-games-security.md [hunk 4]
```diff
@@ -515,34 +499,14 @@ Click the `OktaOAuth2` GameObject and notice the public variables from the `Okta
...
-
-{% img blog/unity-csharp-security/17-okta-clientidsecret.png alt:"Okta add web app" width:"1000" %}{: .center-image }
+From the Okta CLI `.okta.env`, copy the **OKTA_OAUTH2_CLIENT_ID** and **OKTA_OAUTH2_CLIENT_SECRET** into the **Client ID** and **Client Secret** variables in the inspector for the `OktaOAuth2` GameObject in Unity.
 
 {% img blog/unity-csharp-security/18-unity-clientidsecret.png alt:"Okta add web app" width:"1000" %}{: .center-image }
 
-For the three endpoint URL's, go back to Okta. In the top menu click **API** > **Authorization Servers**.
-
-{% img blog/unity-csharp-security/19-okta-authserver.png alt:"Okta add web app" width:"1000" %}{: .center-image }
+For the three endpoint URL's, go back to Okta. In the menu click **API** > **Authorization Servers**.
```
Comment (mraible) [inline _source/_posts/2020-08-21-unity-csharp-games-security.md:506]:
```suggestion
For the three endpoint URLs, go back to Okta (use `okta login` to get a link in your terminal). In the menu click **Security** > **API** > **Authorization Servers**.
```

**Review by mraible — CHANGES_REQUESTED** [review]

File: _source/_posts/2020-08-21-unity-csharp-games-security.md (hunk 1 above)
Comment (nickgamb) [reply _source/_posts/2020-08-21-unity-csharp-games-security.md:52]: removed because they show the old UI and no CLI

Comment (nickgamb) [reply _source/_posts/2020-08-21-unity-csharp-games-security.md]: Added back in after chatting with @mraible in slack.

File: _source/_posts/2020-08-21-unity-csharp-games-security.md [hunk 5]
```diff
@@ -213,8 +212,7 @@ Make sure `MenuManager` is selected and notice the new properties in the `PanelM
 1. Drag the `AuthMenu` object from the project hierarchy to the `authMenu` property in the inspector.
 2. Drag the InputField from the `Username` object in the project hierarchy to the `Username` property in the inspector.
 3. Drag the InputField from the `Password` object in the project hierarchy to the `Password` property in the inspector.
-4. Enter the Okta domain from the Okta developer org "{yourOktaDomain}".
-
+4. Replace `{yourOktaDomain}` with the domain of your Okta developer org.
```
Comment (mraible) [inline _source/_posts/2020-08-21-unity-csharp-games-security.md:215]:
```suggestion
4. Replace `{yourOktaDomain}` with your Okta domain.
```

**Review by mraible — APPROVED** [review]

**Pull request merged**